integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts

```diff
@@ -164,14 +164,14 @@ export class OpfCheckoutBillingAddressFormService {
     this.isSameAsDeliverySub.next(value);
   }
 
-  protected getDeliveryAddress(): Observable<Address | undefined> {
+  getDeliveryAddress(): Observable<Address | undefined> {
```

Comment:
Is the access modifiers were changed only for using this in spec? If yes, i don't think it's a good idea.
I think will be better to access this methods with `service['getDeliveryAddress']()` or to use `service as any` in spy if needed

Reply:
fixed